components/layout.js

```diff
@@ -1,50 +1,56 @@
 import Link from "next/link";
-import { LogoIcon, GitIcon, LinkedinIcon } from "./assets/icons";
+import { LogoIcon } from "./assets/icons";
+import ThemeToggler from "../components/ThemeToggler";
 
 export default function Layout({ children }) {
   return (
-    <div className="flex flex-col min-h-screen bg-[url('/floating-cogs.svg')] bg-opacity-10">
-      <header className="bg-[#316B83] mb-8 py-4 px-2">
-        <div className="container mx-auto flex justify-center">
+    <div className="flex flex-col min-h-screen bg-white dark:bg-black bg-opacity-10">
+      <header className="bg-[#316B83] dark:bg-gray-800 mb-8 py-4 px-2">
```

Comment:
Can we define constant color variables in a global css file and use them in respective files accordingly for better readability ?